fapi-client/src/main/scala/com/gu/facia/api/models/collectionconfig.scala

```diff
@@ -0,0 +1,119 @@
+package com.gu.facia.api.models
+
+import com.gu.facia.client.models.CollectionConfigJson
+
+case class Groups(groups: List[String])
+
+sealed trait DynamicContainerType
+object DynamicFast extends DynamicContainerType
+object DynamicSlow extends DynamicContainerType
+
+sealed trait FixedContainerType
```

Comment:
I'll just change it to a wrapper for the string ID. You're not using these at the moment, and I'd prefer to just have the string available in `frontend`.